src/lib.rs

```diff
@@ -3,6 +3,7 @@
 //! a given target amount.
 //!
 
+#![feature(doc_cfg)]
```

Comment:
Would be better if this was conditionally enabled because if not one cannot run `cargo test` with stable toolchain.
```rust
#![cfg_attr(docsrs, feature(doc_cfg))]
```

Reply:
@tcharding good catch, thanks man.   I've added the conditional in so `cargo test` does not fail on `stable`.

Reply:
@tcharding it looks like `#![cfg_attr(docsrs, feature(doc_cfg))]` causes CI to fail.  I looked at [rust-bitcoin/bitcoin/contribt/test.sh](https://github.com/rust-bitcoin/rust-bitcoin/blob/master/bitcoin/contrib/test.sh) and it's missing the step `cargo build` which is the step that now fails AFAICT.  I'm curious why is this missing from rust-bitcoin?  If `cargo build` is added to rust-bitcoin, will that CI also fail?

Reply:
nm it looks like I had place the `cfg_attr` in the wrong place, although I'm still curious why rust-bitcoin doesn't do a `cargo build` with all features.